cleartune/ea_genvat_cleartune_fastfield.m

```diff
@@ -77,7 +77,7 @@
 
 stimDir = fullfile(options.subj.stimDir, ea_nt(options), stimname);
 ea_mkdir(stimDir);
-filePrefix = ['sub-', options.subj.subjId, '_sim-'];
+filePrefix = ['sub-', options.subj.subjId, '_label-',S.label, '_sim-'];
```

Comment:
label is added in the vta name for cleartune?

Reply:
as we spoke, I will change the position of the key to the end and name from `label` to `stimset`